app/instance.py

```diff
@@ -234,8 +230,29 @@ def track_user():
 # it is important to register them after celery is defined to resolve circular imports
 
 from .api.helpers import tasks
-
 celery = tasks.celery
+celery.autodiscover_tasks(['app'], force=True)
+
+
+# register scheduled jobs
+from app.api.helpers.scheduled_jobs import send_after_event_mail, send_event_fee_notification, \
+    send_event_fee_notification_followup, change_session_state_on_event_completion, \
+    expire_pending_tickets, send_monthly_event_invoice, event_invoices_mark_due, \
+    delete_ticket_holders_no_order_id
+
+
+@celery.on_after_configure.connect
```

Comment:
@iamareebjamal - Because the tasks are not added to the celery as in the file `scheduled-jobs.py` I have imported celery from `instance.py`.

`from app.instance import celery`.

 Hence also to register the task and schedule from the celery instance which is being created by `tasks.py` I have to do this

Reply:
Also the celery instance is created or initiliazed in `tasks.py` not `scheduler jobs.py`

Reply:
You've imported celery instance in scheduled job, hence you can do it in this file only

Reply:
I tried not working, how do we invoke scheduler jobs.py.. That's why I did it here @iamareebjamal

Reply:
You just import it in app.instance